docs/source/examples/pec_tutorial.myst

```diff
@@ -0,0 +1,310 @@
+
+We also define a graph representation of our qubits and assume a line topology.
+
+```{code-cell} ipython3
+# Assume chain-like connectivity
+computer = nx.Graph()
+computer.add_edges_from([(0, 1), (1, 2)])
+
+# Add reversed edges to computer graph.
+# This is important to represent CNOT gates with target and control reversed.
+computer = nx.to_directed(computer)
+
+noisy_backend = LocalSimulator("braket_dm")
+ideal_backend = LocalSimulator("default")
+```
+
+## Define the circuit
+
+We use mirror circuits to benchmark the performance of the device. Mirror circuits, introduced in [arXiv:2008.11294](https://arxiv.org/abs/2008.11294), are designed such that only one bitstring should be sampled. When run on a device, any other measured bitstrings are due to noise. The frequency of the correct bitstring is our target metric.
+
+> Note: Mirror circuits build on Loschmidt echo circuits - i.e., circuits of the form $U U^\dagger$ for some unitary $U$. Loschmidt echo circuits are good benchmarks but have shortcomings - e.g., they are unable to detect coherent errors. Mirror circuits add new features to account for these shortcomings. For more background, see [arXiv:2008.11294](https://arxiv.org/abs/2008.11294).
+
+To define a mirror circuit, we need the device graph. We will use a subgraph of the device, and our first step is picking a subgraph with good qubits.
+
+
+### Generate mirror circuit
+
+Now that we have the device (sub)graph, we can generate a mirror circuit and the bitstring it should sample as follows.
+
+```{code-cell} ipython3
+def get_circuit(depth: int, seed: int) -> Union[Tuple[cirq.Circuit, str], Tuple[qiskit.QuantumCircuit, str]]:
+    circuit, correct_bitstring = benchmarks.generate_mirror_circuit(
+        nlayers=depth,
+        two_qubit_gate_prob=1.0,
+        connectivity_graph=computer,
+        two_qubit_gate_name="CNOT",
+        seed=seed,
+        return_type="braket",
+    )        
+    return circuit, correct_bitstring 
+```
+
+### PEC representations
+
+PEC makes use of quasi-probability representations. We define these here in
+terms of CNOT operations. For more information about these representations in
+the context of PEC and usage within `mitiq`, refer to the ["What is the theory behind PEC"](https://mitiq.readthedocs.io/en/latest/guide/pec-5-theory.html#quasi-probability-representations) page.
+
+```{code-cell} ipython3
+def get_cnot_representation(edge: Tuple[int, int]) -> pec.OperationRepresentation:
+    cnot_circuit = cirq.Circuit(cirq.CNOT(cirq.LineQubit(edge[0]), cirq.LineQubit(edge[1])))              
+    
+    rep_exact_prob = 1 - np.sqrt(1 - cnot_error_prob)
+    return pec.represent_operation_with_local_depolarizing_noise(
+        cnot_circuit, 
+        noise_level=rep_exact_prob,
+    )
+
+
+def get_representations(computer: nx.Graph) -> List[pec.OperationRepresentation]:
+    return [get_cnot_representation(edge) for edge in computer.edges]
+```
+
+### Helper methods
```

Comment:
Do you think these helper functions are useful? It's true that we call them, but their output is never used in this tutorial.
Maybe we can just get rid of them?

Reply:
Hmm. We could. But I think for reproducibility we might want to include them, right?

Reply:
Sounds good